src/AccessibilityInsights.SharedUx/Properties/AutomationIDs.cs

```diff
@@ -13,5 +13,15 @@ public static class AutomationIDs
         public static string TelemetryDialogExitButton { get; } = nameof(TelemetryDialogExitButton);
         public static string EventsDataGrid { get; } = nameof(EventsDataGrid);
         public static string EventModeControl { get; } = nameof(EventModeControl);
+        public static string SettingsButton { get; } = nameof(SettingsButton);
+        public static string SettingsApplicationTabItem { get; } = nameof(SettingsApplicationTabItem);
+        public static string SettingsAboutTabItem { get; } = nameof(SettingsAboutTabItem);
+        public static string ChannelComboBox { get; } = nameof(ChannelComboBox);
```

Comment:
I've been prefixing these with (at least part of) the name of the control/class which contains them. As you mention, we probably want better organization in the future, so consider it a suggestion to use that naming style for now🙂